src/toil/batchSystems/gridengine.py

```diff
@@ -21,9 +21,16 @@
 import logging
 import os
 from pipes import quote
-import subprocess
 import time
 import math
+import sys
+
+# subprocess32 is a backport of python3's subprocess module for use on Python2,
```

Comment:
This is rather verbose, especially considering that this import occurs all over the place.

You can make this a one-liner by using a technique called re-exporting:

In a nutshell, this means placing this code in `src/toil/__init__.py` and then using `from toil import subprocess` wherever `import subprocess` occurs.